_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -38,11 +41,16 @@ The `object` element is not rendered for presentational purposes. If the `object
 
 ## Accessibility Support
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
+Non-supported media formats make screen readers render the text content of the element instead of other attributes. `object` elements without an accessible name are ignored by assistive technologies unless they have an [explicit role][].
 
 ## Background
 
-Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
+This rule is testing that the following 'object' attributes are followed:
+
+- The purpose is in the [accessible name][] but is not part of this rule and must be tested separately; and 
+- When the object resource is not loaded the fallback content is rendered; and
+- Valid `data` defines the [accessible name][] that is announced by the screen reader; and
```

Comment:
I don't think that the `data` attribute of the `object` element plays a role when computing its accessible name. So, I do not understand what you mean by "`data` **defines** the accessible name".